content/tutorial/index.md

```diff
@@ -92,10 +100,10 @@ Let's start by changing **Abell Example** with your name.
 module.exports = {
   // ..
   globalMeta: {
-    siteTitle: 'Saurabh', // replace with your name
-  }
+    siteTitle: "Saurabh", // replace with your name
+  },
   //...
-}
```

Comment:
These changes are formatting changes right? maybe you can revert these. I should've added prettier. Will do that in new version.

Reply:
Thanks for catching that, I reverted this.